src/libsyntax_pos/symbol.rs

```diff
@@ -359,16 +359,20 @@ declare_keywords! {
     (53, Virtual,            "virtual")
     (54, Yield,              "yield")
 
+    // Edition-specific keywords reserved for future use.
+    (55, Async,              "async") // >= 2018 Edition Only
+    (56, Proc,               "proc") // <= 2015 Edition Only
```

Comment:
Why are we keeping proc reserved on 2015? We'll surely never use it for anything on the 2015 epoch.

Reply:
NVM saw the comment explaining this